client/js/libs/handlebars/colorClass.js

```diff
@@ -1,5 +1,3 @@
-"use strict";
-
```

Comment:
eslint complains that "use strict" isn't needed in modules when `parserOptions.sourceType = "module"`. I added a directory-specific eslintrc to client/ which sets that option so the rule doesn't get misapplied to all the non-webpack code (which isn't actually using ES6 modules). https://github.com/eslint/eslint/issues/4936 has some more context.